### DISCUSSION TITLE
Welcome to CoinUtils Discussions!

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/CoinUtils#140] [General]
## 👋 Welcome!
  As you may have heard, Github has been developing a new feature called Discussions, which is now being rolled 
  out in beta to all projects. We envision Discussions as a way of moving the kinds of conversations that currently 
  take place on the old Mailman list serve directly to where development is taking place! To take this thing for a test 
  drive, please post here when you would otherwise post to the list serve. This is a place to connect with other members 
  of our community on topics not suited for opening an Issue. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dlaehnemann** [comment]
I use `cbc` through a wrapper library in Rust, [`rust_lp_modeler`](https://github.com/jcavat/rust-lp-modeler) and have recently reported some input parsing bugs for the `.lp` file format. I found this Discussion forum because I have a question about recommendations for other formats to support.
